pkg/volume/glusterfs/glusterfs.go

```diff
@@ -465,6 +469,8 @@ func (d *glusterfsVolumeDeleter) Delete() error {
 
 func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error) {
 	var err error
+	var reqGid int64
+	gidRandomizer := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
@liggitt as discussed here https://github.com/kubernetes/kubernetes/pull/35460#issuecomment-257071419 , a random GID should satisfy the requirement and iic, the rand source looks to be a common practice we follow in k8s code base.